numpy/lib/function_base.py

```diff
@@ -1482,42 +1482,46 @@ def angle(z, deg=False):
     return a
 
 
-def _unwrap_dispatcher(p, discont=None, axis=None):
+def _unwrap_dispatcher(p, discont=None, axis=None, *, interval_size=2*pi):
```

Comment:
```suggestion
def _unwrap_dispatcher(p, discont=None, axis=None, *, interval_size=None):
```
Dispatcher args should all default to `None`.